.github/workflows/e2e.yml

```diff
@@ -32,8 +32,16 @@ jobs:
         with:
           go-version-file: go.mod
 
+      - name: Check if requested to skip Flatcar tests
+        if: ${{ ! contains( github.event.pull_request.labels.*.name, 'e2e/flatcar') }}
+        run: echo 'SKIP_E2E="Flatcar"' >> "$GITHUB_ENV"
+
+      - name: Check if requested to skip Generic tests
+        if: ${{ contains( github.event.pull_request.labels.*.name, 'e2e/skip-generic') }}
+        run: echo "SKIP_E2E='$SKIP_E2E Generic'" >> "$GITHUB_ENV"
```

Comment:
the delimiter should be `|`

Reply:
the script here would need to check if anything defined in the variable `SKIP_E2E` so add the delimiter `|` or just write `Generic`

Reply:
can you adjust the file then?